core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -72,6 +73,7 @@ describe('Exploration editor page component', function() {
   var stass = null;
   var tds = null;
   var ueps = null;
+  var sampleEmitter = new EventEmitter();
```

Comment:
sampleEmitter is not a good name, can you name it more specific like `refreshGraphEventEmitter` or at least `mockEmitter`?

Reply:
Done!